### PR TITLE
build into a jekyll website

### PR DESCRIPTION
**Pull request by daniellevass**
this is way too much code for 1 commit. I'm so sorry!

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+title: Codebar iOS Workshop Resources
```
Comment (KimberleyCook) [inline _config.yml:16]: codebar is always littlecase c 😛

File: index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+title: Codebar iOS Workshop Resources
```
Comment (KimberleyCook) [inline index.md:2]: again here, lowercase c :D

**Review by KimberleyCook — APPROVED** [review]
2 little comments but they're not blockers :D

**Comment by daniellevass** [comment]
for time purposes - i'll do another PR (hopefully this evening) with the brand guidelines follow up!

**Pull request merged**